test/e2e/index.ts

```diff
@@ -136,3 +136,63 @@ test('error is catchable when holochain socket is unavailable', async (t) => {
     )
   }
 })
+
+
+test('can inject agents', async (t) => {
+    const conductor1 = await launch(ADMIN_PORT, CONFIG_PATH)
+    const conductor2 = await launch(ADMIN_PORT_1, CONFIG_PATH_1)
+    try {
+        const installed_app_id = 'app'
+        const admin1 = await AdminWebsocket.connect(`http://localhost:${ADMIN_PORT}`)
+        const admin2 = await AdminWebsocket.connect(`http://localhost:${ADMIN_PORT_1}`)
+        const agent_key_1 = await admin1.generateAgentPubKey()
+        t.ok(agent_key_1)
+        const agent_key_2 = await admin2.generateAgentPubKey()
+        t.ok(agent_key_2)
+        const nick = 'thedna'
+        let result = await admin1.installApp({
+            installed_app_id, agent_key: agent_key_1, dnas: [
+                {
+                    path: `${FIXTURE_PATH}/test.dna.gz`,
+                    nick,
+                }
+            ]
+        })
+        t.ok(result)
+        const app1_cell  = result.cell_data[0][0]
+        await admin1.activateApp({ installed_app_id })
+        const conductor1_agentInfo = await admin1.requestAgentInfo({cell_id: null});
+        t.equal(conductor1_agentInfo.length, 1)
+        
+        await admin2.addAgentInfo({ agent_infos: conductor1_agentInfo });
+        const conductor2_agentInfo_again = await admin2.requestAgentInfo({cell_id: null});
+        t.equal(conductor2_agentInfo_again.length, 1)
+
+        result = await admin2.installApp({
+            installed_app_id, agent_key: agent_key_2, dnas: [
+                {
+                    path: `${FIXTURE_PATH}/test.dna.gz`,
+                    nick,
+                }
+            ]
+        })
+        t.ok(result)
+        const app2_cell  = result.cell_data[0][0]
+        await admin2.activateApp({ installed_app_id })
+
+        const conductor2_agentInfos = await admin2.requestAgentInfo({cell_id: null});
```

Comment:
What does it mean to pass cell_id: null?
Unfiltered? All peers?

Reply:
It'd be really good to just have some simple comments around the different phases of this test

Reply:
It'd be really good to just have some simple comments around the different phases of this test

Reply:
yes

Reply:
done